examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-5_Privacy_In_Federated_Learning/05.3_homomorphic_encryption/src/kaplan_meier_wf.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
should we use 2025 for codes?

Reply:
Yes, 2025 makes sense as they are new files in this repo. Changed.